scripts/travis-emscripten/build_emscripten.sh

```diff
@@ -43,7 +43,12 @@ fi
 WORKSPACE=/root/project
 
 # Increase nodejs stack size
-sed -i -e 's/NODE_JS="nodejs"/NODE_JS=["nodejs", "--stack_size=8192"]/' /root/.emscripten
+if [ -e ~/.emscripten ]
+then
+    sed -i -e 's/NODE_JS="nodejs"/NODE_JS=["nodejs", "--stack_size=8192"]/' ~/.emscripten
```

Comment:
Should have comments explaining why there is a difference. Also would have squashed the commit :)

Reply:
Oh yeah, you are right...